fastly/computeacl/v1/api_create.go

```diff
@@ -0,0 +1,35 @@
+package v1
```

Comment:
It appears that the API definition doesn't actually include `v1` in the path, so the API designers have not made any provision to support future versions of the API that way.

Since that is the case, I think we can 'lift' all the files from this 'v1' directory into the 'computeacl' directory, and avoid any references to an API version. Sorry for the back-and-forth here, I hadn't realized the API didn't include a version number.

Reply:
No worries! Addressed in 3cafb0abcf9008bb9cb32ff321b68ad76c66f0fb.